lib/bot_commands.rb

```diff
@@ -91,7 +91,7 @@ def self.manage_discord_commands(bot)
                 associated_user = UserAssociatedAccount.find_by(provider_uid: instance[2..19], provider_name: 'discord')
                 if associated_user.nil?
                   discord_username = event.bot.user(instance[2..19]).username
-                  raw = raw.gsub(instance, I18n.t("discord_bot.commands.disccopy.mention_prefix", discord_username: discord_username) + instance[21..])
+                  raw = raw.gsub(instance, "discord_%{discord_username}", discord_username: discord_username + instance[21..])
```

Comment:
Hey there, I'd prefer not to lose the flexibility of a locale string setting here.  We also have a paying client using this plugin, so need to be careful not to disturb their potential overrides.